pallets/automation-time/src/lib.rs

```diff
@@ -1296,6 +1296,9 @@ pub mod pallet {
 			}
 
 			AccountTasks::<T>::remove(task.owner_id.clone(), task_id.clone());
+
+			// Refund reserved execution fees
```

Comment:
No, @imstar15 you need to add a TODO tag to the comment. Otherwise the comment is explaining the code in the line below.

`// TODO: Add refund reserved execution fees here`

Reply:
Fixed